sdk/lusid_drive/utilities/file_streaming.py

```diff
@@ -0,0 +1,47 @@
+import re
+import six
+import requests
+from lusid_drive.utilities.lusid_drive_retry import lusid_drive_retry
+
+
+@lusid_drive_retry
+def stream_file_upload(api_factory, x_lusid_drive_filename, x_lusid_drive_path, content_length, body):
+    headers = {
+        "Accept": "application/json, text/plain, text/json",
```

Comment:
I don't think you need an accept header on the upload, but you might need a `content-type`

check what the C# one users